### PR TITLE
Add example for rb-* recipes

### PR DESCRIPTION
**Pull request by sodre**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by sodre** [comment]
@conda-forge/core,

I would like to add an example recipe for the ruby-packages. Is that something that requires a more formal process or is this PR and request sufficient?

**Comment by CJ-Wright** [comment]
I really like the descriptiveness of the example!

**Comment by isuruf** [comment]
It looks like you can easily write a conda skeleton for this.

**Comment by sodre** [comment]
> It looks like you can easily write a conda skeleton for this.

I thought so as well, but it would probably need to be written in Ruby because their spec file, with dependencies and stuff, is written in ruby based as opposed to json or yaml. 

Another possible route is to scrape the website...

**Comment by isuruf** [comment]
You can use `libraries.io`


https://libraries.io/api/rubygems/json
https://libraries.io/api/rubygems/json/2.2.0/dependencies

**Comment by sodre** [comment]
TIL

**Comment by sodre** [comment]
any thoughts on whether its okay to merge this?

File: recipes/example/ruby/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  # It is sufficient/recommended to copy the summary from the rubygems page itself.
+  summary: A beautiful, minimal theme for Jekyll.
+
+  # The remaining entries in this section are optional, but recommended.
+  doc_url: https://www.rubydoc.info/gems/{{ name }}
+  dev_url: https://github.com/jekyll/{{ name }}
+
+extra:
+  recipe-maintainers:
+    # Feel free to keep me in the recipe-maintainers list for ruby packages.
```
Comment (bgruening) [inline recipes/example/ruby/meta.yaml]: Do we want to establish a ruby group similar to https://github.com/conda-forge/r-sf-feedstock/blob/master/recipe/meta.yaml#L71

Comment (sodre) [reply recipes/example/ruby/meta.yaml]: That would be great. How was that done?

File: recipes/example/ruby/meta.yaml [hunk 2]
```diff
@@ -85,8 +85,8 @@ about:
 
 extra:
   recipe-maintainers:
-    # Feel free to keep me in the recipe-maintainers list for ruby packages.
-    - sodre
+    # Keep the ruby group in the maintainers list
+    - conda-forge/ruby
```
Comment (sodre) [inline recipes/example/ruby/meta.yaml]: @conda-forge/ruby, I would like to have the `ruby` recipes have a common group of maintainers as it was done for the `r` packages. I would like to know if you guys are okay with it (@scopatz, @jimmynguyc)? If not, are you okay with us dropping you from the @conda-forge/ruby team?

**Comment by isuruf** [comment]
> any thoughts on whether its okay to merge this?

I think we should write something similar to conda skeleton cran + cran helper script by @bgruening and let others know to use it. This way, it'll be easier to review them.

**Comment by sodre** [comment]
@isuruf, I don't disagree on writing the skeleton. The issue on my end is just lack of time. 

The example at least gives it a relatively easy place for users to start from.

I have already identified a handful of folks that are creating recipes, and at least one had a mistake #9451 which was caught in time by sheer luck of the Travis being down...

**Pull request closed without merging**